src/pages/Login.tsx

```diff
@@ -24,9 +27,12 @@ const Login = () => {
         <div className="flex items-center justify-between w-full">
           <div className="flex items-center">
             <input
+              id="send-all-button"
               type="checkbox"
-              checked={true}
-              className="checkbox checkbox-accent"
+              className="checkbox 
+                  rounded-md border-2 checkbox-primary"
+              checked={isChecked}
+              onChange={() => setIsChecked(!isChecked)}
```

Comment:
아래처럼 함수형 업데이트를 하는 것을 더 권장한다고 해서 코멘트 남겨봅니다!(상태 업데이트의 안정성 등)

```suggestion
              onChange={() => setIsChecked((prev) => !prev)}
```